minio/thread_pool.py

```diff
@@ -69,22 +74,21 @@ def add_task(self, func, *args, **kargs):
         """ Add a task to the queue """
         self.tasks_queue.put((func, args, kargs))
 
-    def parallel_run(self, func, args_list):
+    def start_parallel(self):
```

Comment:
Now that we don't call add_task in this function, the comment needs to be changed accordingly.

Reply:
> Now that we don't call add_task in this function, the comment needs to be changed accordingly.

Done